### ISSUE TITLE
Discovery start isn't initializing all ganache accounts with ENG tokens

### ISSUE THREAD

**Issue by lauracw100** [enigmampc/dev-help#31]
@lacabra when @levackt was testing my millionaires problem DApp he found an issue where ENG tokens aren't getting minted for 10th ganache account. 

I've been reviewing discovery code to try to find where that happens. It looks like that account is being used by the Principal Node and that might be the reason why it doesn't have any ENG when discovery starts up?

**Comment by levackt** [comment]
For a little background Victor, I highlighted this as an example of errors we can handle in the dapps towards user friendliness, I don't know if it was intentional to have an account that doesn't have ENG, for testing for other reasons.

That said if there's a need for it to match the number of ganache accounts then we should probably wait until the docker env config changes [previously mentioned](https://github.com/enigmampc/dev-help/issues/14) are done. 
Its quite common to start ganache with a lot more accounts for testing, so it'll be convenient in such cases to also have n accounts with ENG to test.

**Comment by lacabra** [comment]
@lauraweindorf: 
This is correct, the 10th account is "reserved" for the KM node, and it gets no tokens. For reference, the code that distributes ENG tokens is [here](https://github.com/enigmampc/enigma-contract/blob/08346f20aad4ff7377a7ff1f737e9a3ab76d0c04/enigma-js/test/Enigma.spec.js#L99), and 4 lines below, you can see how we are excluding the last account (n=10) from the ENG token distribution.

**Comment by lacabra** [comment]
@levackt: Yes, this is something we can configure through ENV vars like the issue you mentioned. Let's leave this open as well, until that is implemented.

**Comment by lauracw100** [comment]
Thanks for that info @lacabra!

**Comment by ainsleys** [comment]
I'm just catching up on our old issues -- is this implemented yet?
